_posts/2024-08-06-how-to-talk-about-CHERI.markdown

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title:  "How to talk to your parents about hardware memory safety"
+date:   2024-08-06
+categories: cheri
+author: David Chisnall
+---
+
+Some conversations are difficult to have with members of older generations who grew up with different social norms.
+In particular, when you're talking to people who grew up with PDP-11s with their completely flat memory, or Lisp machines or Burroughs Large Systems with their deeply opinionated and language-integrated hardware memory safety, you may find it hard to talk about CHERI.
```

Comment:
Is that really the group you need to convince ?  There's not many of us left, and those that are tend to have enough experience of systems changing to get it.  I'd have though the bigger challenge and group to address was those that think they an get memory safety just by writing better code, using a different language, or getting AI to do it.